src/auth/oauth2/gui/qgsauthoauth2edit.ui

```diff
@@ -448,10 +448,13 @@
                <property name="bottomMargin">
                 <number>0</number>
                </property>
+               <item>
+                <widget class="QComboBox" name="comboRedirectHost"/>
```

Comment:
Is there any utility in having the combo box editable so people can enter other values if needed?

Reply:
is that potentially dangerous? If they put in an external url by accident, i mean. 

I'd rather defer that till there's a specific need for it.

Reply:
@nyalldawson , reasonable :)